### ISSUE TITLE
Immagine categoria Home

### ISSUE THREAD

**Issue by ciroslitz** [retepasw/joomla-italiapa#390]
Si richiede la possibilità di poter implementare, se possibile, di rendere visibile la foto di default per le  varie categorie in modo da poterla vedere in home per i singoli articoli, nel caso poi venga caricato per l'articolo  verrebbe caricata quella

**Comment by atodaro** [comment]
Se non ho capito male l'idea sarebbe questa:
Creo una categoria "Example" e attribuisco alla categoria un'immagine che sarà assegnata come "immagine introduttiva" per tutti gli articoli della categoria "example"
Creo un articolo nella categoria "Example": l'articolo sarà visualizzato in un modulo newsflash, layout "focus" direttamente in homepage; nell'articolo, l'immagine introduttiva sarà quella assegnata alla categoria "Example".

Provo a ipotizzare: ci dovrebbe essere un "campo aggiuntivo" tra le opzioni delle Categorie che consenta di caricare un'immagine e attribuirla a Tutti gli articoli di quella categoria, lasciando sempre la possibilità di attribuire al singolo articolo un'immagine differente in caso si voglia differenziare la comunicazione.
Il campo aggiuntivo si potrebbe chiamare "Immagine introduttiva articoli"
![Campi aggiuntivi](https://user-images.githubusercontent.com/12257435/105385533-beca6000-5c13-11eb-8a33-433c4b5338df.png)

**Comment by ciroslitz** [comment]
Esatto, questa è la mia idea, tornerebbe utile per alcune categorie tipo circolari, assemblee sindacali, pon.
